src/global/utils/RandomUtil.java

```diff
@@ -64,4 +66,26 @@ public static RandomClause<?> castRandomClause(Clause clause) {
         if (!(clause instanceof RandomClause)) return null;
         return (RandomClause<?>) clause;
     }
+
+    /**
+     * Determine if a list of frequencies is random
+     *
+     * @param frequencies a list of frequency counts for bins
+     * @return whether or not this array of frequncies describes randomly generated values
+     */
+    public static boolean frequenciesAreRandom(int[] frequencies, int numBins) {
```

Comment:
Do we even need `numbins` as a param here? Isn't the frequency array always going to be of size `numBins`?

Reply:
Yes, tbh idk I was wary of having too many implicit assumptions that someone would have to read comments/already understand to comprehend - so I opted to just pass in the value since it's readily available and already calculated